src/clinical/clinical-entities.ts

```diff
@@ -19,17 +19,8 @@ export interface Donor {
 }
 
 export interface AggregateClinicalInfoStats {
-  submittedCoreFields: number;
-  submittedExtendedFields?: number;
-  expectedCoreFields: number;
-  expectedExtendedFields?: number;
-}
-
-export interface ClinicalInfoStats {
-  submittedCoreFields: number;
-  submittedExtendedFields?: number;
-  expectedCoreFields: number;
-  expectedExtendedFields?: number;
+  coreEntitiesStats: Record<CoreClinicalEntites, number>;
```

Comment:
not to repeat but I noticed  you only filling this with 0/1

Reply:
Specimen can be partial since we want a record for each specimen. So it can be a decimal.

Reply:
I wanted to keep it consistent, also if any of them changes how core is calculated we may have to use numbers for it